Library/Homebrew/language/python.rb

```diff
@@ -177,7 +177,7 @@ def needs_python?(python)
       def virtualenv_install_with_resources(options = {})
         python = options[:using]
         if python.nil?
-          pythons = %w[python python3 python@3 python@3.7 python@3.8 pypy pypy3]
+          pythons = %w[python python3 python@3 python@3.7 python@3.8 python@3.9 pypy pypy3]
```

Comment:
I'm fine with this being merged as-is for now but: can we find a way of doing this that doesn't require updating this hardcoded list with all the versions we're using?

Reply:
```suggestion
          pythons = %w[python python3 pypy pypy3] + Formula.names.select { |name| name.start_with? "python@" }
```

Something like this maybe?

It will ignore `python@3` since it's an alias. If we want to keep it we could either use `%w[python python3 python@3 pypy pypy3]` or add `Formula.aliases.select { |name| name.start_with? "python@" }`

Reply:
Yeh, this makes sense to me 👍🏻.